CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.3]
+
+### Changed
+* Updated IPF version tests for versions >=3.80 and verified compatibility with Sentinel-1C data.
```

Comment:
I may have missed the S1C-related changes in this PR; is that covered by the new test cases that you chose for `tests/test_ipf.py`?